typhos/display.py

```diff
@@ -414,16 +414,22 @@ def _clicked(self):
         self.template_selected.emit(template)
 
     def generate_context_menu(self):
-        """Context menu request."""
+        """Context menu request.
+            Duplicate filenames will show full path. """
```

Comment:
The multi-line docstring here isn't indented correctly, but it could have stayed as a single-liner
```suggestion
        """Context menu request. Duplicate filenames will show full path. """
```